README.md

```diff
@@ -25,7 +25,10 @@ Production: [https://electrofishing-query.ugrc.utah.gov](https://electrofishing-
 
 ### Setup
 
-1. `npm install` from root and functions folders.
-1. `npm run copy:arcgis` to copy the ArcGIS API for JavaScript files to the public folder. This needs to be done anytime the arcgis package is updated.
-1. Create `.env.local` files in root and functions folders using the corresponding `.env` files as examples.
-1. `npm start` to start the development server.
```

Comment:
It would have been easier to just write this yourself rather than asking me to do it and then changing it all. :)

Reply:
What you added was great. I unnecessarily formatted it. Sorry about that.